sql/src/main/java/org/apache/druid/sql/calcite/aggregation/builtin/EarliestLatestAnySqlAggregator.java

```diff
@@ -274,13 +299,31 @@ public Aggregation toDruidAggregation(
             args.size()
         );
     }
-
     return Aggregation.create(
         Collections.singletonList(theAggFactory),
         finalizeAggregations ? new FinalizingFieldAccessPostAggregator(name, aggregatorName) : null
     );
   }
 
+  private Integer getMaxStringBytes(
+      final List<RexNode> rexNodes,
+      final AggregateCall aggregateCall,
+      final PlannerContext plannerContext
```

Comment:
this method seems strange to pass `List<RexNode>`, and only uses `rexNodes.get(1)` from it, please pass the `RexNode` to check directly